src/uniprotkb/components/protein-data-views/VariationView.tsx

```diff
@@ -90,60 +181,89 @@ const VariationView = ({
           <th>Description</th>
-          <th>Disease association</th>
+          <th>
+            Disease
+            <br />
+            association
+          </th>
+          <th>Provenance</th>
         </tr>
       </thead>
       <tbody>
-        {Array.from(transformedData.variants)
-          .sort(sortByLocation)
-          .map((variantFeature) => {
-            let position = variantFeature.start;
-            if (variantFeature.start !== variantFeature.end) {
-              position += `-${variantFeature.end}`;
-            }
-
-            return (
-              <Fragment key={variantFeature.protvistaFeatureId}>
-                <tr
-                  data-id={variantFeature.protvistaFeatureId}
-                  data-start={variantFeature.start}
-                  data-end={variantFeature.end}
-                >
-                  <td>{position}</td>
-                  <td>
-                    {variantFeature.wildType}
-                    {'>'}
-                    {variantFeature.alternativeSequence}
-                  </td>
-                  <td>
-                    {variantFeature.descriptions?.map((description) => (
-                      <div key={description.value}>
-                        {`${description.value} (${description.sources.join(
-                          ', '
-                        )})`}
-                      </div>
-                    ))}
-                  </td>
-                  <td>
-                    {variantFeature.association &&
-                    variantFeature.association.length > 0
-                      ? 'Y'
-                      : 'N'}
-                  </td>
-                </tr>
-                <tr
-                  data-group-for={variantFeature.protvistaFeatureId}
-                  data-start={variantFeature.start}
-                  data-end={variantFeature.end}
-                >
-                  <td>
-                    <div>
-                      <strong>Consequence: </strong>
-                      {variantFeature.consequenceType}
+        {filteredVariants.map((variantFeature) => {
+          let position = variantFeature.start;
+          if (variantFeature.start !== variantFeature.end) {
+            position += `-${variantFeature.end}`;
+          }
+
+          return (
+            <Fragment key={variantFeature.protvistaFeatureId}>
+              <tr
+                data-id={variantFeature.protvistaFeatureId}
+                data-start={variantFeature.start}
+                data-end={variantFeature.end}
+              >
+                <td>
+                  {Array.from(
+                    new Set(variantFeature.xrefs.map((xref) => xref.id))
+                  ).map((id, i) => (
+                    <Fragment key={id}>
+                      {i !== 0 && <br />}
+                      {id}
+                    </Fragment>
+                  ))}
+                </td>
+                <td>{position}</td>
+                <td className={styles.change}>
```

Comment:
I wonder if a tooltip or help document link would be useful to here to explain the various symbols (eg `*`, `-`)

Reply:
Do you think it should be added to the https://beta.uniprot.org/help/variant article? We would need a curator to phrase it correctly from a biological point of view